### PR TITLE
Dice icon name and desc change

### PR DESCRIPTION
**Pull request by Spess-Rando**
It's like #36037 but without heresy.

The icons for results of a d4 are now called stuff like "d4-2" rather than "d42", which could be mistaken for a die with 42 sides.

Also, a d2 is now an actual die.

d2

### CODE REVIEW

**Comment by Qustinnus** [comment]
how does this improve the code though

**Comment by Slignerd** [comment]
>The icons for results of a d20 are now called stuff like "d20-10" rather than "d2010", which could be mistaken for a die with 2010 sides.

**Comment by Slignerd** [comment]
I thought it would be a nice change, but there seems to be just too much controversy around this.

**Comment by praisenarsie** [comment]
```
23:38:35
Qustinnus_
can someone give slig's dice PR the proper tags :^) 
23:38:51
Qustinnus_
theyre tryna slip "improves code" on a text tweak 
```

**Comment by Qustinnus** [comment]
thats better

File: code/game/objects/items/dice.dm [hunk 1]
```diff
@@ -148,8 +148,8 @@
 	return
 
 /obj/item/dice/fourdd6
-	name = "4d d6"
-	desc = "A die that exists in four dimensional space. Properly interpreting them can only be properly done with the help of a mathematician, a physicist, and a priest."
+	name = "4D die"
```
Comment (ChangelingRain) [inline code/game/objects/items/dice.dm]: removes the joke

**Comment by ChangelingRain** [comment]
a d2 is a coin. this new d2 sprite looks nothing like a coin.

**Comment by Slignerd** [comment]
It says "coins are undignified" so I thought it wasn't supposed to be. I based the sprite on this.

image

**Comment by ChangelingRain** [comment]
coins are undignified is another joke, because *it is a coin*

**Comment by Slignerd** [comment]
Goddammit.

**Comment by Wilchenx** [comment]
First i thought it was a non euclidean dice.

then i saw what you based it on and got dissapointed

**Comment by ExcessiveUseOfCobblestone** [comment]
Why did you make a new PR for something you could have edited on the previous branch...?

**Comment by Slignerd** [comment]
Because I did exactly that, and got a meme response when I asked to have the PR opened so that it could update with the changed from the remote, even after I removed the changes that caused it to be closed.

**Comment by ExcessiveUseOfCobblestone** [comment]
stupid slime

**Comment by PKPenguin321** [comment]
awful pr title

**Comment by Slignerd** [comment]
you might think so
but this is /tg/
everyone knows that "d" stands for "dice"

**Comment by ChangelingRain** [comment]
uh, no, not without a number after it we don't.

**Comment by ExcessiveUseOfCobblestone** [comment]
ima give slig the d...20

**Pull request merged**